CONTRIBUTING.md

```diff
@@ -145,6 +145,57 @@ feat(pencil): add 'graphiteWidth' option
 perf(pencil): remove graphiteWidth option
 ```
 
+## Becoming a Maintainer
+
+The documentation for becoming a maintainer has been taken from [Foreman](https://theforeman.org/handbook.html#Becomingamaintainer) and adapted for the PatternFly project.
+
+As a maintainer, it is your responsibility to help manage and maintain the health of the PatternFly project.  A PatternFly maintainer has commit permissions to one or more of the repositories under the [PatternFly organization](https://github.com/patternfly). To see a list of maintainers to the PatternFly project, view the [PatternFly GitHub Teams](https://github.com/orgs/patternfly/teams) page.
+
+Maintainers are members of the PatternFly community who exhibit most of the following behaviors:
+
+- Review and merge code and documentation.
+- Help triaging bugs and testing pull requests.
+- Make well formed pull requests.
+- Have a sense of duty about the project.
+- Play well with others, are respectful, show gratitude.
+
+If you want to become a maintainer, we expect you to:
+
+- Review and test pull requests submitted by others.
+- Encourage and ensure design remains an integral part of the review process and pull in designers for review as needed (you can leverage @patternfly/patternfly-ng-ux if there is no known associated designer).
+- Maintain sustained activity versus sporadic.
+- Support users and other developers on [PatternFly Slack](https://patternfly.slack.com/) (there is a channel dedicated to Patternfly NG) and the [mailing list](mailto:patternfly@redhat.com).
```

Comment:
We already provide the Slack information at the top of this doc.

Reply:
The line is about the responsibility to support these channels as a maintainer vs just providing the the links for those things. 

You can just remove the links so it reads
>Support users and other developers on the PatternFly slack channels and mailing list.